### PR TITLE
query Multiple Tables

### PR DESCRIPTION
**Pull request by Abdulwhbhussain**
Project requirements
Create a table named owners with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
full_name: string
age: integer
Create a table named species with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
name: string
Modify animals table:
Make sure that id is set as autoincremented PRIMARY KEY
Remove column species
Add column species_id which is a foreign key referencing species table
Add column owner_id which is a foreign key referencing the owners table
Remember all this goes in schema.sql file.
Insert the following data into the owners table:
Sam Smith 34 years old.
Jennifer Orwell 19 years old.
Bob 45 years old.
Melody Pond 77 years old.
Dean Winchester 14 years old.
Jodie Whittaker 38 years old.
Insert the following data into the species table:
Pokemon
Digimon
Modify your inserted animals so it includes the species_id value:
If the name ends in "mon" it will be Digimon
All other animals are Pokemon
Modify your inserted animals to include owner information (owner_id):
Sam Smith owns Agumon.
Jennifer Orwell owns Gabumon and Pikachu.
Bob owns Devimon and Plantmon.
Melody Pond owns Charmander, Squirtle, and Blossom.
Dean Winchester owns Angemon and Boarmon.
Remember these insertions and modifications happen in data.sql file.
Write queries (using JOIN) to answer the following questions:
What animals belong to Melody Pond?
List of all animals that are pokemon (their type is Pokemon).
List all owners and their animals, remember to include those that don't own any animal.
How many animals are there per species?
List all Digimon owned by Jennifer Orwell.
List all animals owned by Dean Winchester that haven't tried to escape.
Who owns the most animals?
Remember all these should be written in queries.sql file.
Take a screenshot of the results of your queries.
Remember to add the screenshot with the results of your queries to your Pull Request description.
query1
query2
query8
query4
query5
query6
query7

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -87,4 +87,39 @@ SELECT species,
...
+  JOIN animals AS a ON o.id = a.owner_id
+WHERE o.full_name = 'Melody Pond';
+SELECT a.name
+FROM species AS s
+  JOIN animals AS a ON s.id = a.species_id
+WHERE s.name = 'Pokemon';
+SELECT o.full_name,
+  a.name
+FROM owners AS o
+  LEFT JOIN animals AS a ON o.id = a.owner_id;
```
Comment (Munsa1) [inline queries.sql]: - [ ] This query is supposed to list all owners and their animals, remembering to include those that don't own any animal. The output for this query should also show Bob having two animals, `Devimon` and `Plantmon`, and also show Jennifer Orwell owning `Gabumon` and `Pikachu`, Sam Smith should also own `Agumon` kindly fix this.

Your output
image

Expected output
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Abdulwhbhussain, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
All Animals Assigned Owners Correctly :heavy_check_mark: 
ScreenShots Provided :heavy_check_mark:  

## Required Changes ♻️
_Check the comments under the review._
### Due to insufficient files changed I will leave a comment here
- [ ] Kindly fix the MIT link. You should have a LICENSE.md file in your repository containing the license agreement, this is the file that you should link to your readme file MIT license section as per requirement, and NOT the current link you have there.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-- [ ] Consider summarizing your project description as you have added a lot of information

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Abdulwhbhussain) [reply queries.sql]: Implemented

**Comment by Munsa1** [comment]
- [ ] @Abdulwhbhussain consider updating the screenshot after effecting the change

**Comment by Abdulwhbhussain** [comment]
> * [ ]  @Abdulwhbhussain consider updating the screenshot after effecting the change

Already done!

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Wahab,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on implementing recommended changes!

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**